scopes/explorer/ui/base-component-card/base-component-card.tsx

```diff
@@ -29,6 +36,10 @@ export type BaseComponentCardProps = {
    * true if the component is deprecated
    */
   isDeprecated?: boolean;
+  /**
+   * avatar component
+   */
+  avatar?: ComponentType;
 } & React.HTMLAttributes<HTMLDivElement>;
 
 export function BaseComponentCard({
```

Comment:
please consider remove this and use building-blocks directly

Reply:
done